src/core/Akka/Serialization/Serializer.cs

```diff
@@ -121,7 +121,17 @@ public static int GetSerializerIdentifierFromConfig(Type type, ExtendedActorSyst
         {
```

Comment:
It isn't touching `Identifer` property, it can be only used to assign some value to it.